vue-frontend/src/components/partials/HeaderV2.vue

```diff
@@ -0,0 +1,183 @@
+<template>
+  <div class="nav-container" :style="{ height: navContainerHeight + 'px' }">
+    <nav
+      class="navbar navbar-expand-md navbar-light main-header"
+      :class="{
+        'navbar-sticky': navbarSticky,
+        'navbar-animation': navbarAnimation,
+      }"
+      ref="mainHeader"
+    >
+      <div class="container">
+        <router-link to="/" replace>
+          <div class="navbar-brand">
+            <img
+              :src="headerLogoImage"
+              class="header-logo-image mt-1"
+              alt="canopas-logo"
+            />
+          </div>
+        </router-link>
+        <div class="navbar-collapse">
+          <ul class="navbar-nav ms-auto mb-2 mb-lg-0">
+            <li class="nav-item-margin">
+              <a class="nav-link v2-title-3-text bottom-border" target="_blank"
+                >Home</a
+              >
+            </li>
+            <li class="nav-item-margin v2-title-3-text">
+              <a class="nav-link" :href="careerURL">Career</a>
+            </li>
+            <li class="nav-item-margin v2-title-3-text">
+              <a class="nav-link" target="_blank">Blogs</a>
+            </li>
+            <li class="nav-item-margin v2-title-3-text">
+              <a class="nav-link" target="_blank">Portfolio</a>
+            </li>
+            <li class="nav-item-margin round-btn v2-title-3-text">
+              <a class="anchor-color" target="_blank">Let's Talk</a>
```

Comment:
Add hover effect in Let's Talk button 

-     Button background color white and text black